src/main/java/org/xbill/DNS/ResolverConfig.java

```diff
@@ -261,26 +259,32 @@ else if(line.startsWith("options")) {
 		br.close();
 	}
 	catch (IOException e) {
+		return false;
+	}
+
+	if (lserver.isEmpty()) {
+		return false;
 	}
 
 	configureFromLists(lserver, lsearch);
 	configureNdots(lndots);
+	return true;
 }
 
-private void
+boolean
 findUnix() {
-	findResolvConf("/etc/resolv.conf");
+	return findResolvConf("/etc/resolv.conf");
```

Comment:
Can we pass an InputStream to findResolvConf (and wrap the Files.newInputStream(Paths.get("...")) into an auto-close try)?

Reply:
Yes!  Done in latest commit.